src/components/ec-submenu/ec-submenu.vue

```diff
@@ -65,92 +65,85 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/index';
-@import '../../scss/tools/index';
-
-$ec-submenu-body: $level-3-body-and-headings !default;
-$ec-submenu-hover: $level-4-tech-blue !default;
-$ec-submenu-disabled: $level-6-disabled-lines !default;
+<style>
 
 .ec-submenu {
-  position: relative;
+  @apply tw-relative;
 
   &__header {
     min-width: 100px;
-    margin-bottom: 12px;
-    white-space: nowrap;
-    display: flex;
-    flex-direction: column;
-
-    @include media__from-1024 {
-      flex-direction: row;
-      border-bottom: 1px solid $ec-submenu-disabled;
+
+    @apply tw-mb-12;
+    @apply tw-flex tw-whitespace-no-wrap;
+    @apply tw-flex-col;
+
+    @media (min-width: 1024px) {
```

Comment:
```
@screen lg {
  ...
}
```

Reply:
Do I need to do something more? It seems like is not work only with that